Resources/Prototypes/Recipes/Crafting/Graphs/improvised/shotgunsawn.yml

```diff
@@ -0,0 +1,21 @@
+- type: constructionGraph
+  id: ShotgunSawn
+  start: start
+  graph:
+    - node: start
+      edges:
+        - to: shotgunsawn
+          steps:
+            - prototype: ShotgunDB
+              icon:
+                sprite: Objects/Weapons/Guns/Shotguns/db_shotgun.rsi
+                state: icon
+              name: double-barreled shotgun
+            - prototype: SawElectric
```

Comment:
This should be a tool step if you want the saw to not be consumed.

Define the tool quality for sawing here [Prototypes](https://github.com/space-wizards/space-station-14/tree/master/Resources/Prototypes)/tool_qualities.yml then you can add a tool component to each saw, giving the electric variants a better speed like below:
`  - type: Tool
    qualities:
      - Sawing
    speed: 1.5`